.travis.yml

```diff
@@ -37,6 +37,7 @@ matrix:
     env:
       # Unfortunately, the build fails on MacOs for universal poker.
       - BUILD_WITH_ACPC="OFF"
+      - BUILD_WITH_JULIA="OFF" # untested on mac yet
```

Comment:
Would you like me to try building the Julia wrapper on Mac?

Reply:
That would be great! I don't have a Mac computer at hand so it's not very convenient for me to test it. I guess the `install.sh` needs some modification correspondingly.

Reply:
It seems that CxxWrap has a bug on Mac. https://github.com/JuliaInterop/CxxWrap.jl/issues/182